sdk/formrecognizer/Azure.AI.FormRecognizer/README.md

```diff
@@ -1,136 +1,339 @@
-<!-- 
-If you use the Azure CLI, replace `<your-resource-group-name>`, `<your-resource-name>`, `<location>`, and `<sku>` with your values:
+* An existing Cognitive Services or Form Recognizer resource.
+
+### Create a Cognitive Services or Form Recognizer resource
+Form Recognizer supports both [multi-service and single-service access][cognitive_resource_portal]. Create a Cognitive Services resource if you plan to access multiple cognitive services under a single endpoint/key. For Form Recognizer access only, create a Form Recognizer resource.
+
+You can create either resource using: 
+
+**Option 1:** [Azure Portal][cognitive_resource_portal].
+
+**Option 2:** [Azure CLI][cognitive_resource_cli]. 
+
+
+Below is an example of how you can create a Form Recognizer resource using the CLI:
 
 ```PowerShell
-az cognitiveservices account create --kind TextAnalytics --resource-group <your-resource-group-name> --name <your-resource-name> --location <location> --sku <sku>
-``` -->
-<!-- 
-### Install the package
-Install the Azure Text Analytics client library for .NET with [NuGet][nuget].  To use the [.NET CLI](https://docs.microsoft.com/en-us/nuget/consume-packages/install-use-packages-dotnet-cli), run the following command from PowerShell in the directory that contains your project file:
+# Create a new resource group to hold the form recognizer resource -
+# if using an existing resource group, skip this step
+az group create --name <your-resource-name> --location <location>
+```
 
 ```PowerShell
-dotnet add package Azure.AI.TextAnalytics --version 1.0.0-preview.1
-``` 
-For other installation methods, please see the package information on [NuGet][nuget].
--->
+# Create form recognizer 
+az cognitiveservices account create \
+    --name <your-resource-name> \
+    --resource-group <your-resource-group-name> \
+    --kind FormRecognizer \
+    --sku <sku> \
+    --location <location> \
+    --yes
+```
+For more information about creating the resource or how to get the location and sku information see [here][cognitive_resource_cli].
 
 ### Authenticate a Form Recognizer client
-In order to interact with the Form Recognizer service, you'll need to select either a `ReceiptClient`, `FormLayoutClient`, or `CustomFormClient`, and create an instance of this class.  In the following samples, we will use CustomFormClient as an example.  You will need an **endpoint**, and either an **API key** or ``TokenCredential`` to instantiate a client object.  For more information regarding authenticating with cognitive services, see [Authenticate requests to Azure Cognitive Services][cognitive_auth].
+In order to interact with the Form Recognizer service, you'll need to create an instance of the `FormRecognizerClient`[form_recognizer_client_class] class.  You will need an **endpoint** and an **API key** to instantiate a client object.  
```

Comment:
Having the `` don't work here with the link. consider removing them

Reply:
Updated. Looks like it was missing the [] around the outside of the ``.